apps/docs/src/components/home/HeroSection.tsx

```diff
@@ -8,11 +8,11 @@ const SlideText = () => {
   const slideKeyframes = useMemo(
     () => `
     @keyframes slide {
-      0% { transform: translateY(100%); opacity: 0.1; }
+      0% { transform: translateY(110%); opacity: 0.1; }
```

Comment:
the easter egg is gone 😢 
![image](https://github.com/user-attachments/assets/a801f0b7-048c-4153-ba6e-80d9a21dd3f8)